### DISCUSSION TITLE
Animation from Y = 0 to real value

### DISCUSSION THREAD

**Discussion by matpascaud** [patrykandpatrick/vico#588] [Questions] [answered]
### How to reproduce
Edit: After hours searching in discussion and issues, I found chartView.setDiffAnimationDuration(0)
This can be closed, sorry for that.

Hi,
I am using Vico 2.0.0-alpha.8 with Views.
I am adding a new value to a cartesian chart and would like to avoid the animation where the added point starts from Y = 0 to the expected value.
I have seen diffAnimationSpec but do not see how to use it without using Compose.

### Observed behavior

Here is a video of the "bug" when slowing down the number of added points.

https://github.com/patrykandpatrick/vico/assets/534304/2edcf382-8c40-48f9-8c3d-c7f3de4b0a3a
If I add points at 10Hz, points are not displayed at the correct position (Y > 30) but displayed below the threshold 20.
![Screenshot_20240227_173322](https://github.com/patrykandpatrick/vico/assets/534304/3598e6b7-56ac-412d-9a11-a937283e4033)

### Expected behavior

I expect to see the point at the expected coordinate.

### Vico version(s)

Latest unstable version

### Android version(s)

33

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Calling `CartesianChartView#setDiffAnimationDuration` with an argument of zero disables the animation.

**Comment by matpascaud** [comment]
Using chartView.setDiffAnimationDuration(0) solves the problem.

**Reply by patrickmichalik** [reply]
I’m glad to hear that!
